package.json

```diff
@@ -13,56 +13,41 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-intl": "^4.7.6",
     "@alteryx/icons": "^1.0.0",
     "@alteryx/react-comms": "^1.0.0",
-    "@alteryx/ui": "^1.0.0",
+    "@alteryx/ui": "^1.0.0"
+  },
+  "devDependencies": {
     "@babel/cli": "7.11.6",
     "@babel/core": "7.11.6",
+    "case-sensitive-paths-webpack-plugin": "2.3.0",
+    "clean-webpack-plugin": "3.0.0",
+    "compression-webpack-plugin": "4.0.1",
+    "file-loader": "6.1.0",
+    "fork-ts-checker-webpack-plugin": "5.2.0",
+    "html-webpack-plugin": "4.5.0",
+    "mini-css-extract-plugin": "0.11.2",
     "@babel/preset-env": "7.11.5",
     "@babel/preset-react": "7.10.4",
     "@babel/preset-typescript": "^7.13.0",
     "@babel/runtime": "^7.14.0",
-    "@types/enzyme": "3.10.7",
-    "@types/enzyme-adapter-react-16": "1.0.6",
-    "@types/jest": "26.0.14",
     "@types/node": "11.15.27",
     "@types/react": "16.9.49",
     "@types/react-dom": "16.9.8",
     "babel-loader": "8.1.0",
-    "case-sensitive-paths-webpack-plugin": "2.3.0",
-    "clean-webpack-plugin": "3.0.0",
-    "compression-webpack-plugin": "4.0.1",
-    "css-loader": "3.6.0",
-    "dotenv": "8.2.0",
-    "file-loader": "6.1.0",
-    "fork-ts-checker-webpack-plugin": "5.2.0",
-    "html-webpack-plugin": "4.5.0",
-    "mini-css-extract-plugin": "0.11.2",
-    "optimize-css-assets-webpack-plugin": "5.0.4",
-    "react-hot-loader": "4.13.0",
-    "rimraf": "^3.0.2",
-    "style-loader": "1.2.1",
-    "ts-loader": "8.0.4",
-    "typescript": "3.9.7",
-    "webpack": "4.44.2",
-    "webpack-bundle-analyzer": "3.9.0",
-    "webpack-cli": "3.3.12",
-    "webpack-dev-server": "3.11.0",
-    "webpack-merge": "4.2.2"
-  },
-  "devDependencies": {
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/plugin-transform-runtime": "^7.13.15",
     "babel-plugin-transform-imports": "^2.0.0",
     "core-js": "^3.11.2",
-    "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.6",
-    "enzyme-react-intl": "^2.0.6",
-    "enzyme-to-json": "^3.6.2",
-    "fs-extra": "9.0.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-intl": "^4.7.6",
-    "transform-imports": "^2.1.1"
+    "rimraf": "^3.0.2",
+    "transform-imports": "^2.1.1",
+    "webpack": "4.44.2",
```

Comment:
If this dependency audit is meant to update our package dependencies, then we should consider migrating to `webpack@5` and `typescript@4`.

Reply:
It wasn't originally around doing things like major updates, more just a clean up.